runtime_builders/publish_runtime.py

```diff
@@ -28,23 +29,45 @@
 def main():
     logging.getLogger().setLevel(logging.INFO)
     parser = argparse.ArgumentParser()
-    parser.add_argument('--infile', '-i',
+    parser.add_argument('--directory', '-d',
                         help='templated cloudbuild config file.',
                         required=True)
     parser.add_argument('--bucket', '-b',
                         help='GCS bucket to publish runtime to',
                         default='runtime-builders')
-    parser.add_argument('--builder-name',
-                        help='the name of the runtime or project '
-                        'associated with this builder',
-                        required=True)
     args = parser.parse_args()
 
-    templated_file_contents = _resolve_tags(args.infile)
-    logging.info(templated_file_contents)
-    logging.info(_publish_to_gcs(templated_file_contents,
-                                 args.builder_name,
-                                 args.bucket))
+    return _resolve(args.directory, args.bucket)
+
+
+def _resolve(directory, bucket):
+    try:
+        gcs_paths = []
+        for filename in os.listdir(directory):
```

Comment:
You might be able to replace this with the 'glob' module.

```python
import glob
glob.glob(os.path.join(directory, "*.json"))
```
or something similar.